src/shared/main/cpp/native_c4replicator.cc

```diff
@@ -422,6 +422,7 @@ JNIEXPORT jlong JNICALL Java_com_couchbase_lite_internal_core_C4Replicator_creat
     C4ReplicatorParameters params = {};
     params.push = (C4ReplicatorMode) jpush;
     params.pull = (C4ReplicatorMode) jpull;
+    params.dontStart = true;
```

Comment:
Agree with Blake's point here. cc @snej

Reply:
It was for backward compatibility, to avoid breaking all the clients with that commit. Default behavior (assuming they zero-fill the struct) remains the same.